### PR TITLE
Added reloadContent() to LeanRooster

### PR DESCRIPTION
**Pull request by adriantran**
Added reloadContent() to LeanRooster; also fixed FocusOutShellSample to call OfficeFabric API initializeIcons() per documentation so all icons show up.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "license": "MIT",
   "dependencies": {
     "@microsoft/load-themed-styles": "1.7.14",
-    "office-ui-fabric-react": "~5.12.0",
+    "office-ui-fabric-react": "~5.79.0",
     "react": "~15.3.2",
     "react-dom": "~15.3.2",
     "roosterjs-editor-plugins": "^6.9.2",
```
Comment (pranavgk) [inline package.json]: Can we also bump this up?

Comment (adriantran) [reply package.json]: not required (atm)

File: sample/FocusOutShellSample.htm [hunk 2]
```diff
@@ -20,7 +20,7 @@
                 overflow: auto;
             }
             .lean-rooster.view-mode + .rooster-command-bar {
-                display: none;
+                /*display: none;*/
```
Comment (pranavgk) [inline sample/FocusOutShellSample.htm]: Undo or remove?

Comment (adriantran) [reply sample/FocusOutShellSample.htm]: was debugging, undoing...

**Pull request merged**